File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/marketingvb/vb/distributecampaignfrommarketinglist9.vb
' <snippetdistributecampaignfrommarketinglist9>


	' Qualify the marketing list.
          Dim qualifyRequest =
              New QualifyMemberListRequest With
              {
                  .OverrideorRemove = OverrideOrRemove.Override,
                  .MembersId = {_contactIdList(0), _contactIdList(1)},
                  .ListId = _copiedMarketingListId
              }

	_serviceProxy.Execute(qualifyRequest)

          Console.WriteLine("  Qualified the copied marketing list so that it only" _
                            &amp; vbCrLf &amp; "    includes the first two members.")

' </snippetdistributecampaignfrommarketinglist9>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/marketing/cs/distributecampaignfrommarketinglist9.cs
// <snippetdistributecampaignfrommarketinglist9>


 // Qualify the marketing list.
 var qualifyRequest = new QualifyMemberListRequest
 {
     OverrideorRemove = OverrideOrRemove.Override,
     MembersId = new[] { _contactIdList[0], _contactIdList[1] },
     ListId = _copiedMarketingListId
 };

 _serviceProxy.Execute(qualifyRequest);

 Console.WriteLine("  Qualified the copied marketing list so that it only\r\n    includes the first two members.");

// </snippetdistributecampaignfrommarketinglist9>